scripts/genpinctrl/stm32-pinctrl-config.yaml

```diff
@@ -247,4 +247,3 @@
 
 - name: USB
   match: "^USB_(?:DM)?(?:DP)?(?:NOE)?$"
-
```

Comment:
Nitpicking: I think you should discard the change on this last file line.

Reply:
It should not have been added and was removed by my editor. I'll keep it that way.